torch/_decomp/decompositions.py

```diff
@@ -1634,7 +1630,7 @@ def maybe_mask(vals, length, range_max, adaptive, dim):
     return ret / (length_h * length_w)
 
 
-@register_decomposition(aten.index_add_)
+@register_decomposition(aten.index_add_, disable_meta=True)
```

Comment:
It would be good to have some commentary about why we switched this. IIRC, @Chillee said it's because `index_add_` decomp is wrong?